docs/features/logforwarding/outputs/azure-monitor-log-forwarding.adoc

```diff
@@ -0,0 +1,165 @@
+=== Steps to forward to Azure Monitor Log via HTTP Data Collector API
+
+(https://learn.microsoft.com/en-us/azure/azure-monitor/logs/data-collector-api?tabs=powershell)
+
+. Create a secret containing your `shared_key` using the following command (primary or the secondary key for the workspace that's making the request):
++
+----
+ oc create secret generic azure-secret -n openshift-logging --from-literal=shared_key='<shared_key>'
+----
+
+. Create a Cluster Logging instance with collection `type: vector`:
```

Comment:
Let's exclude this since it is only supported in vector and this is not required as part of mCLF

Reply:
ok